public/js/logix.js

```diff
@@ -1454,14 +1453,13 @@ CircuitElement.prototype.isVerilogResolvable = function() {
     return res;
 }
 
-CircuitElement.prototype.removePropogation = function() {
+CircuitElement.prototype.removePropagation = function() {
     for (var i = 0; i < this.nodeList.length; i++) {
-        if (this.nodeList[i].type == NODE_OUTPUT) {
-            if (this.nodeList[i].value !== undefined) {
-                this.nodeList[i].value = undefined;
-                simulationArea.simulationQueue.add(this.nodeList[i]);
-            }
-        }
+        var node = this.nodeList[i];
+        if (node.type !== NODE_OUTPUT) continue;
+        if (node.value !== undefined) continue;
+        node.value = undefined;
+        simulationArea.simulationQueue.add(node);
```

Comment:
Just remove code refactoring from the PR, rest looks fine

Reply:
We still want different PRs for code and comments right? @tachyons

Reply:
Fixed @tachyons

Reply:
It is fine as long as it is just fixing typos. Should not contain any changes in logic or code, also when fixing typo in a method name, all the instances of that method call is updated

Reply:
Yep, I've reverted my changes to the code. I suppose I'll address it in a separate PR. As to the method names, I have done a manual check and changed all instances of the `removePropagation` method. In addition, all commits have been squashed